velox/dwio/parquet/tests/ParquetTpchTest.cpp

```diff
@@ -123,163 +146,173 @@ class ParquetTpchTest : public testing::Test {
     return splits;
   }
 
-  // Write the DuckDB Lineitem TPC-H table to a Parquet file and return the file
-  // location.
-  std::string writeLineitemTableToParquet() const {
-    constexpr std::string_view tableName("lineitem");
-    constexpr int kRowGroupSize = 10'000;
-    const auto& filePath =
-        fmt::format("{}/{}.parquet", tempDirectory_->path, tableName);
-    // Convert decimal columns to double.
-    const auto& query = fmt::format(
-        "COPY (SELECT l_orderkey as orderkey, l_partkey as partkey, l_suppkey as suppkey, l_linenumber as linenumber, "
-        "l_quantity::DOUBLE as quantity, l_extendedprice::DOUBLE as extendedprice, l_discount::DOUBLE as discount, "
-        "l_tax::DOUBLE as tax, l_returnflag as returnflag, l_linestatus as linestatus, "
-        "l_shipdate as shipdate, l_receiptdate as receiptdate, "
-        "l_shipinstruct as shipinstruct, l_shipmode as shipmode, l_comment as comment "
-        "FROM {}) TO '{}' (FORMAT 'parquet', ROW_GROUP_SIZE {})",
-        tableName,
-        filePath,
-        kRowGroupSize);
-    duckDb_->execute(query);
-    return filePath;
+  std::shared_ptr<const RowType> getRowType(
+      const std::string& table,
```

Comment:
consistency: writeTableToParquet takes std::string_view, but this method takes std::string. It would be nice to choose one of these and use them consistently.